purchases/src/main/kotlin/com/revenuecat/purchases/PostReceiptHelper.kt

```diff
@@ -104,10 +107,8 @@ internal class PostReceiptHelper(
             marketplace = purchase.marketplace,
             initiationSource = initiationSource,
             onSuccess = { postReceiptResponse ->
-                // Currently we only support a single token per postReceipt call but multiple product Ids.
-                // The backend would fail if given more than one product id (multiline purchases which are
-                // not supported) so it's safe to pickup the first one.
-                // We would need to refactor this if/when we support multiple tokens per call.
+                // Currently we only support a single token per postReceipt call but multiple product Ids
+                // (for multi-line subscriptions).
```

Comment:
For the acknowledgement logic, I guess if we acknowledge the purchase that includes the add-ons, that will acknowledge all the subscriptions, correct?

Reply:
yes it's the same purchase token